homeassistant/components/mqtt/image.py

```diff
@@ -0,0 +1,199 @@
+"""Image that loads a picture from using an MQTT topic."""
```

Comment:
Can we make this more readable, maybe:
```suggestion
"""Support for MQTT images."""
```